src/app/[locale]/incident/add/components/IncidentQuestionsLocationForm.tsx

```diff
@@ -33,6 +36,12 @@ export const IncidentQuestionsLocationForm = () => {
           formStoreState.sub_category
         )
 
+        console.log(
+          additionalQuestions,
+          formStoreState.main_category,
+          formStoreState.sub_category
+        )
+
```

Comment:
Zijn deze console.log statements de bedoeling? Misschien is het handig om dit met no-console in Eslint te verbieden, zodat je een expliciete beslissing over console.log kunt duidelijk maken met // eslint-ignore-next-line